beacon-chain/rpc/prysm/v1alpha1/validator/duties_test.go

```diff
@@ -507,8 +695,11 @@ func BenchmarkCommitteeAssignment(b *testing.B) {
 		Epoch:      0,
 	}
 	b.ResetTimer()
+
 	for i := 0; i < b.N; i++ {
 		_, err := vs.GetDuties(context.Background(), req)
 		assert.NoError(b, err)
+		_, err = vs.GetDutiesV2(context.Background(), req)
+		assert.NoError(b, err)
```

Comment:
This is a benchmark and I dont think it makes sense to use both methods here. I would just update it to the new one and report the difference (if any)

Reply:
Actually, i think this benchmark is broken and doesn't work anyway. Let's just leave it alone and delete or fix in another PR